.circleci/scripts/build_announce.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euox
+
+VERSION=$(go run build/rpm/main.go version)~${CIRCLE_TAG##*-}
+
+SHORT_SHA1=${CIRCLE_SHA1:0:7}
+CIRCLE_PR_NUMBER=$(hub pr list -s open -L 10 -f "%I")
+
+PACKAGES="kcn kpn ken kscn kspn ksen kbn kgen homi"
+BAOBAB_PACKAGES="kcn kpn ken"
+#PACKAGE_PREFIX="http://packages.klaytn.net/klaytn/${VERSION}"
+PACKAGE_PREFIX="http://package-dev.klaytn.net/klaytn/${VERSION}"
+
+LINUX_PACKAGE_LINKS=""
+DARWIN_PACKAGE_LINKS=""
+BAOBAB_LINUX_PACKAGE_LINKS=""
+BAOBAB_DARWIN_PACKAGE_LINKS=""
+
+for i in ${PACKAGES}
+do
+    LINUX_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-linux-amd64.tar.gz">$i</a>"
+    LINUX_PACKAGE_LINKS+=" "
+    DARWIN_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-darwin-10.10-amd64.tar.gz">$i</a>"
+    DARWIN_PACKAGE_LINKS+=" "
+done
+
+for i in ${BAOBAB_PACKAGES}
+do
+    BAOBAB_LINUX_PACKAGE_LINKS+="<a href="${PACKAGE_PREFIX}/$i-${VERSION}-0-linux-amd64.tar.gz">$i</a>"
```

Comment:
Does this tar file contains `k*nd` script and `k*nd.conf` file? 
If it does, you may need to change `PACKAGE_PREFIX` to another prefix for baobab files.

Reply:
Thanks. I just added commit to add baobab prefix on their links. It has `k*nd, k*nd.conf` as you said and link should be like this. 
http://package.klaytn.net/klaytn/v1.5.0~rc.2/kcn-baobab-v1.5.0~rc.2-0-linux-amd64.tar.gz